exampleservice/CustomerService/Handler/CheckSessionHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using exampleservice.CustomerService.Contract;
+using exampleservice.CustomerService.Controller;
+using exampleservice.CustomerService.Events;
+using exampleservice.Framework.Abstract;
+using exampleservice.Framework.BaseFramework;
+using System;
+using System.Threading.Tasks;
+
+namespace exampleservice.CustomerService.Handler
+{
+    internal class CheckSessionHandler : CustomerHandlerBase<CheckSessionCommand>
+    {
+        private IMessageBus bus;
+        private ICustomerServiceDataBaseRepository dataBaseRepository;
+
+        public CheckSessionHandler(IMessageBus bus, ICustomerServiceDataBaseRepository dataBaseRepository)
+        {
+            this.bus = bus ?? throw new ArgumentNullException(nameof(bus));
+            this.dataBaseRepository = dataBaseRepository ?? throw new ArgumentNullException(nameof(dataBaseRepository));
+        }
+
+        internal override async Task<EventBase> Handle(CheckSessionCommand command)
+        {
+            // 1. Session Valid?
+            // 2. Renew Session (if necessary)
+
+            this.VerifyIputArguments(command);
+
+
```

Comment:
are that consecutive new-lines good practice in C# (real question, I don't know C# code-style that much)

Reply:
not really - only if it helps for reading the code (in a complex logic) - in this case its just 👎